src/utils/internal/parseGraphQLRequest.ts

```diff
@@ -157,11 +157,9 @@ export function parseGraphQLRequest(
 
     // Encountered a matching GraphQL request that is syntactically invalid.
     // We may consider getting the parsing error and propagating it to the user.
-    console.error(
+    throw new Error(
```

Comment:
Could you please adjust the tests for `src/utils/internal/parseGraphQLRequest.test.ts` to reflect this change?